ui/src/data_explorer/components/QueryMaker.js

```diff
@@ -10,7 +10,7 @@ const rawTextBinder = (links, id, action) => text =>
   action(links.queries, id, text)
 
 const buildText = (q, timeRange) =>
-  q.rawText || buildInfluxQLQuery(q.range || timeRange, q) || ''
+  q.rawText || buildInfluxQLQuery(timeRange, q) || ''
 
```

Comment:
Agreed as well, I was worried about that until I checked and discovered it being completely separate due to TVs being used on Dashboards.